test/generator.ml

```diff
@@ -0,0 +1,59 @@
+open Messages
+
+module type MTest = sig
```

Comment:
Do we need this? Can't we just expose everything?

Reply:
As you want. I found it was overrated to have a dedicated .mli file, but maybe too confusing to don't expose only the `signatures`...
So as you wish, I'm ok for all 😅